SearchAggregator.sln

```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.24720.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "SearchAggregator", "src\SearchAggregator\SearchAggregator.xproj", "{1086DCD1-50C1-4A86-89EE-7ECB92C2B43F}"
```

Comment:
SearchAggregator proxy service won't be merged into dev. It used to be a prototyping playground. Will submit a cleaner PR changes before merging.